src/components/avatar/__stories__/index.story.jsx

```diff
@@ -0,0 +1,130 @@
+import * as React from 'react';
+import { Text, Avatar } from '@/index';
+
+export default {
+  title: 'Components/Avatar',
+  component: Avatar,
+  parameters: {
+    docs: {
+      docPage: {
+        a11yProps: ` 
```

Comment:
Add description for avatar component

Reply:
Avatars are used to identify users through display pictures or their initials.